src/components/pages/geographyOriginalPage.tsx

```diff
@@ -217,6 +221,48 @@ export const GeographyOriginalPage = ({ geography, summary, targets, theme, them
     }
   };
 
+  /** Vespa search results */
+  const vespaSearchTabbedNavItems = themeConfig.categories
+    ? themeConfig.categories.options.map((category) => {
+        return {
+          title: category.label,
+          /** We need to maintain the slug to to know what to send to Vespa for querying. */
+          slug: category.slug,
+          // TODO: Make this work
+          count: 0,
+        };
+      })
+    : /** We generate an `All` for when themeConfig.categories are not available e.g. MCFs */
+      [
+        {
+          title: "All",
+          slug: undefined,
+          // TODO: Make this work
+          count: 0,
+        },
+      ];
+  const [currentVespaSearchSelectedCategory, setCurrentVespaSearchSelectedCategory] = useState(vespaSearchTabbedNavItems[0].title);
```

Comment:
The default starting category is also stored within the config under the attribute `defaultDocumentCategory`. The intention here was to ensure this category is selected by default when navigating to the geo page. For the CPR app for instance, this is "Laws"